h5p-bildetema-words-grid-view/src/components/Image/Image.module.scss

```
@@ -1,4 +1,6 @@
 .img {
+  box-sizing: border-box;
+  padding: 1px;
```

Could perhaps only add padding to left and right, since it is only needed there (?)

```suggestion
  padding: 0 1px;
```